tests/core/test_fileio.py

```diff
@@ -6,14 +6,20 @@
 from ansys.meshing.prime import ErrorCode, PrimeRuntimeError
 
 
-def test_io_file_not_found_error(get_remote_client):
+def test_io_file_not_found_error(get_remote_client, get_examples):
     model = get_remote_client.model
     with prime.FileIO(model) as io:
         file_name = r'/doesnotexist/file.pmdat'
         with pytest.raises(FileNotFoundError) as error:
             _ = io.read_pmdat(file_name, prime.FileReadParams(model=model))
             assert f'Given file name "{file_name}" is not found on local disk' in str(error.value)
 
+        files = [r'/doesnotexist/file.pmdat', get_examples["elbow_lucid"]]
+        with pytest.raises(FileNotFoundError) as error:
```

Comment:
It would be nice to check the error message too.

```suggestion
        error_msg = "File /doesnotexist/file.pmdat  given for read is missing from local disk."
        with pytest.raises(FileNotFoundError, match=error_msg) as error:
```